jerry-core/parser/regexp/re-parser.c

```diff
@@ -262,7 +262,10 @@ re_count_num_of_groups (re_parser_ctx_t *parser_ctx_p) /**< RegExp parser contex
     {
       case LIT_CHAR_BACKSLASH:
       {
-        lit_utf8_incr (&curr_p);
+        if (curr_p != parser_ctx_p->input_end_p)
```

Comment:
The LIT_CHAR_LEFT_PAREN case below uses < instead of !=. Is there a reason for a difference? < may be a bit safer than !=.

Reply:
There is no particular reason for using `!=` instead of `<` because both ways inflict the same behaviour.
I can change it if you like.

Reply:
If by any mistake curr_p jumps over input_end_p, < catches it but != does not. I'd suggest the change.